components/address-input.jsx

```diff
@@ -1,6 +1,256 @@
 import React from 'react';
 import IntlMixin from 'react-intl';
 
+var CountrySelect = React.createClass({
```

Comment:
I'm just making CountrySelect a reusable thing internally. Mean copy and pasting a semi-large chunk of code.